### ISSUE TITLE
Action fails to parse form

### ISSUE THREAD

**Issue by CollinHeist** [edumserrano/github-issue-forms-parser#56]
Using the following as the form body (copy-pasted directly from the issue)

---

### Series Name

BNN

### Series Year

2013

### Blueprint Description

Tinted Frame card using the title Font (Impact) in the title yellow color.

### Blueprint File

```json
{
  "series": {
    "font_id": 0,
    "card_type": "tinted frame",
    "translations": [],
    "season_title_ranges": [],
    "season_title_values": [],
    "extra_keys": [],
    "extra_values": [],
    "template_ids": []
  },
  "episodes": {},
  "templates": [],
  "fonts": [
    {
      "name": "BNN",
      "color": "rgb(255, 249, 98)",
      "delete_missing": true,
      "file": "impact.ttf",
      "interline_spacing": -20,
      "size": 1.2,
      "vertical_shift": -10
    }
  ],
  "preview": "preview.jpg"
}
```

### Zip of Files

[Archive.zip](https://github.com/CollinHeist/TitleCardMaker-Blueprints/files/12071859/Archive.zip)

---

The parser fails, saying:

> Option -i|--issue-body expects a single argument, but provided with multiple:

---

The associated workflow step is this:

      # Parse issue into JSON
      - name: Parse issue
        id: issue_to_json
        uses: edumserrano/github-issue-forms-parser@v1.3.3
        with:
          template-filepath: '.github/ISSUE_TEMPLATE/new_blueprint.yml'
          issue-form-body: '${{ github.event.issue.body }}'

**Comment by edumserrano** [comment]
I believe I've fixed this with #57. I've also released a [new version of the action](https://github.com/edumserrano/github-issue-forms-parser/releases/tag/v1.3.4) so you should be able to test the fix.

I'm no longer able to replicate the problem so I'm closing this issue. However, if you still face problems let me know and we can re-open this.

## Note

You can change the line:
```
uses: edumserrano/github-issue-forms-parser@v1.3.3
```

to
```
uses: edumserrano/github-issue-forms-parser@v1.3.4
```

or change it to:
```
uses: edumserrano/github-issue-forms-parser@v1
```

If you leave it as `v1` then it will pick up the latest possible version under `v1` which means that if this gets updated tomorrow you will automatically get the updated. If you use `v1.3.4` then you're locking yourself to that version and you'll have to manually update whenever you want.

**Comment by edumserrano** [comment]
This also led me to find another bug in the app which I resolved with #58. I recommend using the `v1` tag or at least use `v1.3.5` to get the fix.

**Comment by CollinHeist** [comment]
Perfect, thanks for the quick fix. v1.3.5 seems to work for my needs.

Thanks again!
